sdk/src/client/node_api/core/mod.rs

```diff
@@ -49,7 +46,7 @@ impl Client {
 
     /// Request outputs by their output ID in parallel, ignoring failed requests
     /// Useful to get data about spent outputs, that might not be pruned yet
-    pub async fn try_get_outputs(&self, output_ids: Vec<OutputId>) -> Result<Vec<OutputWithMetadataResponse>> {
+    pub async fn try_get_outputs(&self, output_ids: Vec<OutputId>) -> Result<Vec<OutputWithMetadata>> {
```

Comment:
@Alexandcoats proposed to rename it to something like `get_outputs_ignore_xxx` or something with `_unchecked`